src/client/components/MegaMenu/MegaMenu.module.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../styles/_global.scss';
+
+.menu {
+  @include centered-block;
+  padding: 4rem 0 6rem;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+
+  :global(.ams-icon){
+    display: none;
+  }
+
+  & > :nth-child(3) :global(.ams-link-list__link) {
+    font-weight: 800;
+  }
+
+  @include mq-phone() {
+    grid-template-columns: 1fr;
+
+    & > :nth-child(2) {
+      order: 3;
+    }
+
+    & > :nth-child(4) {
+      order: 4;
+    }
+  }
+
+  & > div {
+    padding-left: unset;
+    padding-right: unset;
+  }
+
+  .ams-mega-menu > div {
+    margin-left: unset;
+    margin-right: unset;
+  }
```

Comment:
Doet dit iets? Ik zie deze class niet terug in de react `className` props.

Reply:
hehe ja die was nieter nodig na @import '@amsterdam/design-system-css/src/components/logo/logo.scss';